services/image-puller/Makefile

```diff
@@ -0,0 +1,12 @@
+include ../../scripts/common.Makefile
+include ../../scripts/common-service.Makefile
+
+APP_NAME := $(notdir $(CURDIR))
+
+.DEFAULT_GOAL := help
+
+
+.env: .env-devel ## creates .env file from defaults in .env-devel
+	$(if $(wildcard $@), \
```

Comment:
did you forget to commit ``.env-devel`` ???

Reply:
added one